Assets/Source/Entities/Behaviors/Influential.cs

```diff
@@ -1,14 +1,33 @@
 using UnityEngine;
 using System.Collections;
+using System.Collections.Generic;
 
 public class Influential : MonoBehaviour
 {
+	private static HashSet<Influential> queens = new HashSet<Influential>();
+	public static HashSet<Influential> Queens {
```

Comment:
This may be a special case though, particularly because these properties are static. That's why I separated them. Do you think visibility should override static in terms of formatting? Or the other way around? I don't really mind either way, but that's why I did it the way I did.